public/app/core/components/PageNew/SectionNavItem.tsx

```diff
@@ -0,0 +1,118 @@
+import { useStyles2, Icon, toIconName } from '@grafana/ui';
+
+export interface Props {
+  item: NavModelItem;
+}
+
+export function SectionNavItem({ item }: Props) {
+  const styles = useStyles2(getStyles);
+
+  const children = item.children?.filter((x) => !x.hideFromTabs);
+  const isRoot = item.parentItem == null;
+  const hasChildren = Boolean(children?.length);
+
+  // If first root child is a section skip the bottom margin (as sections have top margin already)
+  const noRootMargin = isRoot && Boolean(item.children![0].children?.length);
+
+  const linkClass = cx({
+    [styles.link]: true,
+    [styles.activeStyle]: item.active,
+    [styles.isSection]: hasChildren,
+    [styles.isRoot]: isRoot,
+    [styles.noRootMargin]: noRootMargin,
+  });
+
+  return (
+    <>
+      <a
+        href={item.url}
+        className={linkClass}
+        aria-label={selectors.components.Tab.title(item.text)}
+        role="tab"
+        aria-selected={item.active}
+      >
+        {isRoot && item.icon && <Icon name={toIconName(item.icon)!} />}
+        {isRoot && item.img && <img className={styles.sectionImg} src={item.img} alt={`logo of ${item.text}`} />}
+        {/* {!isRoot && <div className={styles.iconSpacer} />} */}
+        {item.text}
+        {item.tabSuffix && <item.tabSuffix className={styles.suffix} />}
+      </a>
+      {children?.map((child, index) => (
+        <SectionNavItem item={child} key={index} />
+      ))}
+    </>
+  );
+}
+
+const getStyles = (theme: GrafanaTheme2) => {
+  const brandColor = colorManipulator.alpha(theme.v1.palette.brandPrimary, 0.08);
+  const activeBg = theme.isLight
+    ? `linear-gradient(90deg, ${brandColor} 10%, ${theme.colors.action.disabledBackground} 50%)`
+    : `linear-gradient(90deg, ${brandColor} 10%, ${theme.colors.action.disabledBackground} 50%)`;
+
+  return {
+    link: css`
+      padding: ${theme.spacing(1, 0, 1, 1.5)};
+      display: flex;
+      align-items: center;
+      gap: ${theme.spacing(1)};
+      height: 100%;
+      cursor: pointer;
+      position: relative;
+      color: ${theme.colors.text.secondary};
+
+      &:hover,
+      &:focus {
+        text-decoration: underline;
+      }
+    `,
+    activeStyle: css`
+      label: activeTabStyle;
+      color: ${theme.colors.text.primary};
+      /* background-color: ${theme.colors.action.disabledBackground}; */
+      background: ${activeBg};
+      border-radius: ${theme.shape.borderRadius(2)};
+      font-weight: 500;
+
+      &::before {
+        display: block;
+        content: ' ';
+        position: absolute;
+        left: 0;
+        width: 4px;
+        bottom: 2px;
+        top: 2px;
+        border-radius: 2px;
+        background-image: linear-gradient(0deg, #f05a28 30%, #fbca0a 99%);
+      }
+    `,
+    suffix: css`
+      margin-left: ${theme.spacing(1)};
+    `,
+    sectionImg: css({
+      height: 18,
+    }),
+    iconSpacer: css({
+      width: 26,
+    }),
```

Comment:
are either of these used anymore? i don't think they'll do anything without units 🤔